sorts/comb_sort.py

```diff
@@ -1,8 +1,13 @@
 """
+This is pure python implementation of comb sort algorithm.
```

Comment:
python --> Python  # it is a brand name, not a snake.

Reply:
Is it even needed to say its implemented in Python given that this entire repository should just be python algorithms?

Reply:
I think it is fine to write that this is a Python implementation.
The code might be copied elsewhere outside this repo.